inspirehep/pidstore/minters/base.py

```diff
@@ -32,6 +32,11 @@ def validate(self):
     def get_pid_values(self):
         return get_value(self.data, self.pid_value_path, default=None)
 
+    @property
+    def pid_value(self):
+        """Returns pid_value - Required by InvenioRecordsREST POST view"""
+        return self.get_pid_values()
```

Comment:
this might return array

Reply:
In case it will, we expect the code to blow up with an exception.